pom.xml

```diff
@@ -43,6 +43,7 @@
         <testcontainers.version>1.17.6</testcontainers.version>
         <tjenestespesifikasjoner.version>2612.db4dc68</tjenestespesifikasjoner.version>
         <kafka.version>3.4.0</kafka.version>
+        <spring-kafka-test.version>3.0.5</spring-kafka-test.version>
```

Comment:
Tror denne versjonen er for spring 6, som tas i bruk i spring boot 3
Spring boot 3 er ikke tatt inn i tilbake ennå pga en avhengighet som ikke er støttet ennå med jakarta https://github.com/veraPDF/veraPDF-library/issues/1314

Reply:
Hva er det som ikke virker med nåværende versjon? :)

Reply:
Virker som at det er denne som stiller til trøbbel `<kafka.version>3.4.0</kafka.version>`
Spring ellers bruker 3.1.2

Reply:
Mitt forslag er å fjerne hele version fra kafka-client og la Spring styre versjonen

Reply:
Samme kan også gjøres for postgresql dependency. Begge disse er managed gjennom Spring boot